easybuild/easyconfigs/g/git-lfs/git-lfs-2.7.1.eb

```diff
@@ -0,0 +1,31 @@
+# @author: jg (cscs)
+easyblock = "Tarball"
+
+name = 'git-lfs'
+version = '2.7.1'
+
+homepage = 'https://git-lfs.github.com/'
+description = """Git Large File Storage (LFS) replaces large files such as
+audio samples, videos, datasets, and graphics with text pointers inside Git,
+while storing the file contents on a remote server like GitHub.com or GitHub
+Enterprise."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://github.com/%(name)s/%(name)s/releases/download/v%(version)s/']
+sources = ['%(name)s-linux-amd64-v%(version)s.tar.gz']
+
+sanity_check_paths = {
+    'files': ["git-lfs"],
+    'dirs': [""]
+}
+
+# add the installation dir to PATH
+modextrapaths = {
+    'PATH': '',
```

Comment:
Isn't something missing here?

Reply:
no because there is no bin in the install dir hence i just need `PATH=$installdir` instead of `PATH=$installdir/bin`